src/main/resources/application-postgre.yaml

```diff
@@ -0,0 +1,19 @@
+server:
```

Comment:
You have this property in application.yaml. Spring always takes properties from application.yaml and then it looks at active profile. So you don't need to duplicate it in application-h2 and appliaction-postgre

Reply:
I understand, thank you